frontend/src/components/Form/Input.jsx

```diff
@@ -1,22 +1,12 @@
 import React from 'react';
-import CheckBox from './CheckBox';
-import Radio from './Radio';
 import TextInput from './TextInput';
-import Select from './Select';
 import SearchableSelect from './SearchableSelect';
 
 const Input = props => {
   switch (props.type) {
-    case 'checkbox':
-      return <CheckBox {...props} />;
-    case 'select':
-      return <Select {...props} />;
     case 'searchableSelect':
       return <SearchableSelect {...props} />;
```

Comment:
I think you can directly remove this file and use the SearchableSelect directly at the one place it's used